library/src/main/java/com/novoda/downloadmanager/lib/DownloadReceiver.java

```diff
@@ -165,7 +165,7 @@ private void hideNotification(Context context, long batchId) {
                 if ((DownloadStatus.isCancelled(status) || DownloadStatus.isCompleted(status))
                         && (visibility == ACTIVE_OR_COMPLETE || visibility == ONLY_WHEN_COMPLETE)) {
                     ContentValues values = new ContentValues(1);
-                    values.put(DownloadContract.Batches.COLUMN_VISIBILITY, NotificationVisibility.ONLY_WHEN_ACTIVE);
```

Comment:
`ONLY_WHEN_ACTIVE` will only show when the download is active, for completed downloads either can be used, but I think `HIDDEN` is easier to understand :+1: